docker/px4-dev/Dockerfile_ros-kinetic

```diff
@@ -39,6 +40,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 	&& apt-get clean autoclean \
 	# pip
 	&& pip install --upgrade numpy px4tools pymavlink \
+	&& pip3 install --upgrade numpy px4tools pymavlink \
```

Comment:
Do you depend on python3 version of pymavlink?

Also, I'd recommend removing the explicit install of numpy. If you look at the `setup.py` for px4tools your're covered there. https://github.com/dronecrew/px4tools/blob/master/setup.py#L50

I say that because the sole reason for this is ekf processing with px4tools and these dependencies aren't used anywhere else. So, keep it all grounded in px4tools.

Reply:
@lamping7 thank you. as @dagar mentioned I only need pyulog and pyulog also depends on numpy. I'll update the package list.

Reply:
:+1:  numpy is included here https://github.com/PX4/pyulog/blob/master/setup.py#L53

You could probably remove numpy from the pip line as well.